src/resolvers/descriptionResolver.test.ts

```diff
@@ -0,0 +1,56 @@
+import descriptionResolver from "./descriptionResolver"
+import { warning } from '@actions/core'
+
+jest.mock('@actions/core')
+
+interface MockContext {
+  body: string
+}
+
+const makeContext = ({ body = '' }: Partial<MockContext> = {}) => ({
```

Comment:
Could reuse util functions in `src/util/testUtil.ts` for this and also body text generation.